scripts/collect_eval_results.py

```diff
@@ -0,0 +1,103 @@
+"""
+Usage example:
+python scripts/collect_eval_results.py \
+    --experiment_id 01HTNY22Z9YJZ6HDQBF182BGE6 \
+    --job_suffix _v0-step_1.5T-warmup_true-steps_50B_04052024 \
+    --output_file metrics.json
+"""
+
+import beaker
+from pathlib import Path
+import argparse
+from copy import deepcopy
+import json
+
+
+def make_parser():
+    parser = argparse.ArgumentParser(
+        description="""Point this script at a Beaker job created by `submit_eval_jobs.py`.
+                    It will will collect all evaluation metrics and dump them in a json
+                    file. It will also collect summary metrics for each task."""
+    )
+    parser.add_argument(
+        "--experiment_id",
+        type=str,
+        help="ID of the Beaker experiment for which to collect metrics.",
+    )
+    parser.add_argument(
+        "--job_suffix",
+        type=str,
+        help="Suffix of job name; remove this from the job name to get the eval task.",
+    )
+    parser.add_argument(
+        "--output_file", type=Path, help="Output file for collected results."
+    )
+
+    return parser
+
+
+def get_summary_metric(metrics, eval_task):
+    # TODO we need summary metrics for XSTest and AlpacaEval
```

Comment:
@hamishivi what should I use for XSTest and AlpacaEval?

Reply:
XSTest: gpt4_label. refusal_prf.f1 (using dot notation for nested dicts)
AlpacaEval: win_rate (although I need to test what this is called for AlpacaEval2)